controls/cis_sle12.yml

```
@@ -1013,7 +1013,9 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    status: manual # rule is missing / difficult to automate
+    status: automated
+    rules:
+    - set_iptables_outband_n_established_connections_configured
```

```suggestion
    - set_iptables_outbound_n_established_connections_configured
```
You could also choose a more compact name like `iptables_connections_configured`

---

replaced the wrong outband with outbound in  [785b073](https://github.com/ComplianceAsCode/content/pull/10126/commits/785b073bdca7423b98e0f382a163eac64b8cbccc)
It is tempting to have the rule shorter, but kind of prefer to be more specific, in order to avoid future risk of ambiguity

---

I also feel this rule has too long a name. Could it be abbreviated without losing meaning?
Looking to the rule description it is actually referring NEW and ESTABLISHED states in the INPUT and OUTPUT chains. So, the current rule name is also not accurate. Basically here I understood the intention is to allow any OUTPUT connection and their respective flows to return.